src/components/form/form.types.ts

```diff
@@ -229,4 +229,13 @@ export enum FormLayoutType {
      * Render the form fields using a responsive grid layout
      */
     Grid = 'grid',
+
+    /**
+     * Render the form fields in full-width rows.
+     * Each row can have a leading `icon`, and a field.
```

Comment:
There's no indication here on how to provide an icon. We should probably add a type for it, just as we have done for the `GridLayoutOptions`

Reply:
Do you mean we add something like this:
```tsx
export interface RowLayoutOptions
    extends FormLayoutOptions<FormLayoutType.Row> {
    /**
     * gldkjgdlf
     */
    icon?: string;
}
```

But the icon is only for each field/row. Not for the whole form with `row` layout. Do we need to write something like this?

Reply:
Oh, yeah maybe this was something else :smile: 
But I think we had something that could be applied to each item when using the grid layout :thinking:

Reply:
The `rowSpan` and `colSpan` are used on the components when using the `GridLayoutOptions`, but we have documented in there how it should be used :slightly_smiling_face:

Reply:
Yes and no :smile: 
If we can add a type for it I think we should, and we did for the grid options :slightly_smiling_face:

Reply:
so are you requiring changes now or what? 😄 I added a new fixup, but don't know if we should do more or not?

Reply:
I think @Befkadu1 is working on it